bcgov/moh-prime
# PRIME-1882 PIP-Initial-Pass

The starting point of PIP WebAPI
To test:
- Create docker image, run `docker run -t -i --rm pip /bin/ash` in the pip-dotnet-webapi directory
- Create the docker containers, run `docker compose up` in the root of moh-prime, you should see 2 new containers under moh-prime (pipdb, and pip-dotnet-webapi)

For local development
- Make sure you install .NET 6 from [here ](https://dotnet.microsoft.com/download/dotnet/6.0)
- Create a new DB Connection in Dbeaver, same info as prime db, but port is 5433
